runtime/lua/vim/lsp/util.lua

```diff
@@ -768,34 +768,42 @@ function M.convert_input_to_markdown_lines(input, contents)
   return contents
 end
 
+---@param offset integer
+---@param contents string[]
+---@return Range2
+local function get_line_and_col_from_offset(offset, contents)
```

Comment:
```suggestion
local function get_pos_from_offset(offset, contents)
```

p.s.: another case for https://github.com/neovim/neovim/issues/25509 , probably